lib/storage/drbd.py

```diff
@@ -315,6 +315,13 @@ def _GetShowInfo(self, minor):
     """
     return self._show_info_cls.GetDevInfo(self._GetShowData(minor))
 
+  @staticmethod
+  def _NeedsLocalSyncerParams():
+    # For DRBD >= 8.4, syncer init must be done after local, not in net.
+    info = DRBD8.GetProcInfo()
+    version = info.GetVersion()
+    return version["k_minor"] >= 4
```

Comment:
Additionally checking that `version["k_major"] == 8` would make the code more robust.